src/openassetio-core/include/openassetio/hostAPI/Manager.hpp

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+#pragma once
+
+#include <openassetio/export.h>
+#include <openassetio/managerAPI/ManagerInterface.hpp>
+#include <openassetio/typedefs.hpp>
+
+namespace openassetio {
+inline namespace OPENASSETIO_VERSION {
+/**
+ This namespace contains code relevant to anyone wanting to add support
+ for a host application.
+
+ If you are a asset management system developer, see @ref managerAPI.
+*/
+namespace hostAPI {
+/**
+ * The Manager is the Host facing representation of an @ref
+ * asset_management_system. The Manager class shouldn't be directly
+ * constructed by the host.  An instance of the class for any given
+ * asset management system can be retrieved from an API @needsref
+ * Session, using the @ref
+ * openassetio.hostAPI.Session.Session.currentManager
+ * "Session.currentManager" method, after configuring the session with
+ * the appropriate manager @ref identifier.
+ *
+ * @code
+ * session = openassetio.hostAPI.Session(
+ *     hostImpl, consoleLogger, pluginFactory)
+ * session.useManager("org.openassetio.test")
+ * manager = session.currentManager()
+ * @endcode
+ *
+ * A Manager instance is the single point of interaction with an asset
+ * management system. It provides methods to uniquely identify the
+ * underlying implementation, querying and resolving @ref
+ * entity_reference "entity references" and publishing new data.
+ *
+ * The Manager API is threadsafe and can be called from multiple
+ * threads concurrently.
+ */
+class OPENASSETIO_CORE_EXPORT Manager {
+ public:
+  explicit Manager(managerAPI::ManagerInterfacePtr managerInterface);
+
+  /**
+   * @name Asset Management System Information
+   *
+   * These functions provide general information about the @ref
+   * asset_management_system itself. These can all be called before
+   * @needsref initialize has been called.
+   *
+   * @{
+   */
+
+  /**
+   * Returns an identifier to uniquely identify the Manager.
+   *
+   * This identifier is used with the Session class to select which
+   * Manager to initialize, and so can be used as in preferences etc...
+   * to persist the chosen Manager. The identifier will use only
+   * alpha-numeric characters and '.', '_' or '-'. They generally follow
```

Comment:
I've made https://github.com/TheFoundryVisionmongers/OpenAssetIO/issues/361 to track validating comments such as this.

Reply:
What is the problem with this comment, is it that we should allow any UTF-8?

Reply:
Yeah, I think, back in the day we said that as it was a "simpler string to persist", but given we are utf-8 everywhere, do we need to be so ASCII/English centric?